_pages/integrations/influxdb.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: article
+title:  "InfluxDB"
+permalink: integrations/influxdb/
+excerpt: "Monitor model performance metrics with InfluxDB"
+excerpt-short: "Monitor model performance metrics with InfluxDB"
+categories: [integrations]
+tags: [integrations]
+show_related: false
+image:
+    teaser: /language_logos/influxdata.svg
+---
+
```

Comment:
Lets add the enterprise only banner here like on Insights page https://algorithmia.com/developers/algorithmia-enterprise/algorithmia-insights